org.envirocar.app/src/org/envirocar/app/views/trackdetails/MapExpandedActivity.java

```diff
@@ -130,12 +122,11 @@ protected void onCreate(Bundle savedInstanceState) {
         legendEnd = binding.legendEnd;
         legendName = binding.legendUnit;
 
+        // TODO(alexmercerind): Switch to camera API in feature/location-indicator.
         mMapViewExpanded.setOnTouchListener((v, event) -> onTouchMapView());
```

Comment:
I added a new [`CameraState` API](https://github.com/alexmercerind/enviroCar-app/blob/a5b8f8a326cfd451766f957bd80e44a53ffc64ca/org.envirocar.map/src/main/java/org/envirocar/map/MapController.kt#L50-L51) to listen to camera updates.

The existing `setOnTouchListener` callback is not working with new `org.envirocar.map` module. It will be a good idea to switch to the new API for checking whether the default camera state has been changed.

Reply:
I think you have to make sure that the onTouch event will be effective for your custom MapView. Of course, then you can use your `CameraState` API. But up to now, the onTouch event does not reach the MapView.

You have to override the `dispatchTouchEvent` emthod within your custom `MapView` to pass the touch event to the parent view, such as described [here](https://stackoverflow.com/a/17978891).